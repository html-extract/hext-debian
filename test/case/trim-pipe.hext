<a class:trim(" "):' class ' @text:trim:trim("st"):"trimmed content" />
