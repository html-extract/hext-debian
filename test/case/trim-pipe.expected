{" class ":"no spaces","trimmed content":"ome more conten"}
{" class ":"spaces","trimmed content":"This is some conten"}
